media/doc/README.WINE

```diff
@@ -270,6 +270,7 @@ gdi32 -
 
 iphlpapi -
   dll/win32/iphlpapi/icmp.c # Synced to WineStaging-1.7.55
+  modules/rostests/winetests/iphlpapi # Synced to WineStaging-1.9.11
```

Comment:
"Unrelated" question:
am I allowed to WINESYNC iphlpapi icmp.c and related winetest, or must that be done by a dev?

Reply:
Nobody is prohibited from doing that. But any reviewer would need to check your changes, compare them to the WINE changes, and also decide whether those WINE changes make sense for ReactOS.
If the changes amount to just a few lines, a PR may be worth it, but generally a Winesync PR is more work for a reviewer than just doing the Winesync.